meta-mender-core/recipes-mender/mender-connect/mender-connect.inc

```diff
@@ -20,7 +20,13 @@ inherit systemd
 
 GO_IMPORT = "github.com/mendersoftware/mender-connect"
 
+# Skip text relocation warnings
+INSANE_SKIP:${PN} = "textrel"
```

Comment:
Have you looked a bit into the root of this issue? By looking at the [go.bbclass recipe](https://git.yoctoproject.org/poky/tree/meta/classes-recipe/go.bbclass?h=scarthgap) you can see that they pass the `-buildmode=pie` to exactly address this.

My understanding is that this is working fine on the pure go recipes, like `mender-snapshot` below, but not on go code that links with C code, like `mender-connect` and (legacy) `mender-client`.

Although you could say that we are already skipping this check in other recipes, I think it is worth spending a bit more time trying to fix the issue(s) rather than skipping the check.

Reply:
Yes, pie fixes the text relocations, but I didn't get why that wasn't done for the go client, so I just copied the skip.

Reply:
Yes, we already have the skip in other recipes so this is a suggestion, not a requirement :)